_packages/@karrotmarket/gatsby-theme-team-website/src/layouts/DefaultLayout.tsx

```diff
@@ -90,7 +92,10 @@ const DefaultLayout: React.FC<DefaultLayoutProps> = ({
         <html
           lang={data.site.siteMetadata.locale}
           prefix="og: https://ogp.me/ns/website#"
+          data-seed="light dark"
```

Comment:
이거 딱히 포맷이 있진 않아요 ㅋㅋㅋㅋㅋ

```suggestion
          data-seed
```

Reply:
아하 일단 가이드 보고 해놨었는데 없군요ㅋㅋㅋ